reference/component-versions.md

```diff
@@ -5,6 +5,67 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, they are marked with `n/a` below.
 
+## 1.23.0
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | ? |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | ? |
+| `coturn` | `1.23/stable` | ? |
+| `ams` | `1.23/stable` | ? |
+| `aar` | `1.23/stable` | ? |
+| `anbox-stream-gateway` | `1.23/stable` | ? |
+| `ams-lxd` | `1.23/stable` | ? |
+| `anbox-stream-agent` | `1.23/stable` | ? |
+| `nats ` | `latest/stable` | 11 |
+
+#### Ubuntu 20.04 (Deprecated)
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | ? |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | ? |
+| `coturn` | `1.23/stable` | ? |
+| `ams` | `1.23/stable` | ? |
+| `aar` | `1.23/stable` | ? |
+| `anbox-stream-gateway` | `1.23/stable` | ? |
+| `ams-lxd` | `1.23/stable` | ? |
+| `anbox-stream-agent` | `1.23/stable` | ? |
+| `nats ` | `latest/stable` | 11 |
+
+### Bundles
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud` | `1.23/stable` | ? |
+| `anbox-cloud-core` | `1.23/stable` | ? |
+
+### Snaps
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.23/stable` | 1.23.0 |
+| `aar`    | `1.23/stable` | 1.23.0 |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | 1.23.0 |
+| `anbox-cloud-dashboard` | `1.23/stable` | 1.23.0 |
+| `anbox-stream-agent` | `1.23/stable` | 1.23.0 |
+| `anbox-stream-gateway` | `1.23/stable` | 1.23.0 |
```

Comment:
Let's add `anbox-connect` and `anbox-cloud-appliance` here as well.

Reply:
There is no `1.23/stable` channel for the anbox-connect snap; only `latest/stable` is available, similar to `amc` (which also only has a `latest` channel available). @morphis, would it be worth mentioning `amc` here as well, given that `anbox-connect` is mentioned?

Reply:
Fixed it for anbox-connect. For AMC, let me wait for confirmation.

Reply:
Added AMC as well as per SU discussion